dask/dataframe/tests/test_dataframe.py

```diff
@@ -1,3 +1,6 @@
+import pytest
+pytest.importorskip('partd')
```

Comment:
These should be more fine grained around operations that include shuffle operations (`groupby.apply` or `join` on columns).  

Alternatively we could just depend on `partd` explicitly.  Actually, looking at the `setup.py` file it looks like we do depend on it explicitly for `pip install dask[dataframe]`